docs/guide/setup/creating_ligand_networks.rst

```diff
@@ -3,4 +3,68 @@
 Defining the Ligand Network
 ===========================
 
-issue #758
\ No newline at end of file
+A ligand network is a planning unit that we need to plan how we want to calculaute the final free energy network graph.
```

Comment:
could expand here...  I would give a technical description of what it is followed by relating it to concepts that readers are likely already familiar with

Something like "a ligand network is a set of small molecules connected by mappings.  this can therefore represent the enumeration of a given lead compound"

Reply:
tried to improve here.